packages/is-shallow-equal/package.json

```
@@ -25,7 +25,9 @@
 		"build-types",
 		"*.md"
 	],
-	"main": "lib/index.js",
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"react-native": "build/index",
```

Should this refer to `src/index` like e.g. a11y does?

https://github.com/WordPress/gutenberg/blob/2296afb65651f67d132e3c770c9372653e1575dc/packages/a11y/package.json#L25

```suggestion
	"react-native": "src/index",
```